Src/library/net/network/socket/network_stream_socket.e

```diff
@@ -79,30 +81,30 @@ feature -- Initialization
 			end
 		end
 
+	make_server_by_address_and_port (a_address: INET_ADDRESS; a_port: INTEGER)
```

Comment:
No explicitly but I can try, which issue do you suspect?
This will help me for the test.